core/src/com/unciv/ui/worldscreen/unit/UnitTable.kt

```diff
@@ -81,6 +70,20 @@ class UnitTable(val worldScreen: WorldScreen) : Table(){
                     worldScreen.mapHolder.setCenterPosition(it, false, false)
                 }
             }
+
+            addSeparator().actor!!
```

Comment:
the .actor is unnecessary

Reply:
ok

Reply:
In line 64, there is a similar expression assigned to a `separator`. 
- If I copy that line verbatim, Android Studio will say it can't be reassigned.
- On the other hand, not assigning `separator` AS complains at line 48 that `Property must be initialized or be abstract`. 

Is that the white divider line that separates unit label and promotions?

Reply:
Yes.
I don't remember why we save the separator TBH, but it makes sense that it can't be reassigned.
We probably use it somewhere else in the code, but we only need it once.